yb-voyager/cmd/exportSchema_test.go

```diff
@@ -75,4 +93,9 @@ func TestShardingRecommendations(t *testing.T) {
 	modifiedSqlStmt, matchTable, _ = applyShardingRecommendationIfMatching(&sqlInfo_table2, []string{"m1_notfound"}, TABLE)
 	assert.Equal(t, modifiedSqlStmt, "create table a (a int, b int) WITH (fillfactor=70)")
```

Comment:
similarly here, you can can avoid manually writing it again and use sqlInfo_table2.stmt

Reply:
Done.